src/main/java/com/vmtapp/enterprise/dao/ITicketRetrofitDao.java

```diff
@@ -3,7 +3,6 @@
 import com.vmtapp.enterprise.dto.Ticket;
 import retrofit2.Call;
 import retrofit2.http.GET;
-
```

Comment:
https://www.baeldung.com/java-clean-code
Indentation this just keep it more clean and free up any unused space